Currency Converter 1.0

pseudopilot opened the pull request.

Arkant committed:
finish markup, add routing

Commit:
feature/edit-profile
1) change styles profile/edit-profile,
2)  make edit-profile/profile  more flexible

Arkant committed:
add icon, update button sign up style

Arkant committed:
add validation, add wrong log/pass enter, add sign-in method and redirect

pseudopilot committed:
Merge pull request #41 from Arkant/feature/sign-in

Feature/sign in

Commit:
feature/edit-profile
1) add select for ava and styles for it
2) add function createFormInfo to user-profile.service.js
3) start to work on function onFileChanged (edit-profile.controller)

Commit:
feature/edit-profile
add ability to save currentUserInfo to firebase

Commit:
Merge branch 'development' of github.com:Arkant/AngularJSMiniApp into feature/edit-profile

ElenaBorisenkoO committed:
Merge pull request #42 from Arkant/feature/edit-profile

Feature/edit profile

PodstDO committed:
fix/mocks, use $rootScope without recourse to the service usersMocksService

Commit:
feature/edit-profile,
delete alert-message required

Commit:
Merge branch 'development' of github.com:Arkant/AngularJSMiniApp into feature/edit-profile

davidstoff committed:
update/admin, delete input which show user password in admin panel

Alexandra1003 committed:
feature/home
*update getting current user deals

Alexandra1003 committed:
Merge branch 'development' of github.com:Arkant/AngularJSMiniApp into feature/home

update to current changes

pseudopilot committed:
feature/sign-up,

* add methods getProfileImageRef and uploadProfileImage

Arkant committed:
Merge pull request #43 from Arkant/fix/mocks

fix/mocks

Arkant committed:
Merge pull request #44 from Arkant/update/admin

update/admin, delete input which show user password in admin panel

Alexandra1003 committed:
feature/home

*add preloader

Alexandra1003 committed:
Merge branch 'development' of github.com:Arkant/AngularJSMiniApp into feature/home

PodstDO committed:
Merge pull request #45 from Arkant/feature/home

Feature/home

Commit:
feature/edit-profile,
change variable currentUserInfo to currentUser

Commit:
feature/edit-profile,
save changes after merge conflict

ElenaBorisenkoO committed:
Merge pull request #46 from Arkant/feature/edit-profile

Feature/edit profile

davidkin committed:
udpate/adminm, delete extra table point and add move to user profile

Arkant committed:
Merge pull request #47 from Arkant/update/admin

udpate/admin, delete extra table point and add move to user profile

pseudopilot committed:
feature/sing-up,

* improve authentification methods
* add signOut method
* change mocks to rootScpe
* add set picture to profile service

pseudopilot committed:
feature/sign-up,

* change userProfile.ava from array to link

PodstDO committed:
feature/weather, create component:
- create styles file
- create template
- create initial controller
- create component
- create component module
- append component to the HOME page

Alexandra1003 committed:
feature/home
*fix preloading
*add redirection if the user is not signed in

Alexandra1003 committed:
Merge branch 'development' of github.com:Arkant/AngularJSMiniApp into feature/home

ElenaBorisenkoO committed:
Merge pull request #48 from Arkant/feature/home

Feature/home

ElenaBorisenkoO committed:
Merge pull request #50 from Arkant/feature/sign-up

Feature/sign up

Commit:
feature/edit-profile,
add default src to images

Commit:
feature/edit-profile,
fix merge conflict

davidkin committed:
update/admin, delete extra attributes and show point from firebase

davidkin committed:
update/admin, make listOfUsers empty

davidkin committed:
update/admin, delete extra methods and add Users to "listOfUsers" from base

davidkin committed:
update/user-profile.service, fix eslint errors

davidkin committed:
update/sync-data.service, fix eslint errors

davidkin committed:
update/authentication.service, add currentUserId

davidkin committed:
update/converter, rename userDeals -> userDeal, delete value swap

pseudopilot committed:
Merge pull request #51 from Arkant/update/admin

update/admin

Commit:
feature/edit-profile,
fix merge conflict

davidkin committed:
update/user.mock.service, delete extra variables

PodstDO committed:
feature/weather, add weatherAPI service:
- add constant service for component
- use variables in template from controller
- add getData method to weatherAPI service

davidkin committed:
Merge pull request #52 from Arkant/update/user.mocks.service

update/user.mock.service, delete extra variables

Commit:
feature/edit-profile
1) delete currentUser from user.mocks.service
2) add checks to HeaderController and edit-profile.controller
3) add default values for profile.html

Commit:
feature/edit-profile,
fix merge conflict

Commit:
feature/edit-profile,
fix merge conflict

ElenaBorisenkoO committed:
Merge pull request #53 from Arkant/feature/edit-profile

Feature/edit profile

Commit:
feature/edit-profile,
1) change this.firstName/this.lastName to this.currentUser (HeaderController)
2) change saveUserInfoToFirebase (sync-data-service)
3) change function createFormInfo
4) add function saveToCurrentUser

pseudopilot committed:
feature/sign-up,

* delete some extra comments
* add method for changing user role
* correct method for setting profile image

pseudopilot committed:
Merge pull request #54 from Arkant/feature/edit-profile

Feature/edit profile

ElenaBorisenkoO committed:
Merge pull request #55 from Arkant/feature/sign-up

feature/sign-up,

PodstDO committed:
feature/weather, fix merge conflict

Arkant committed:
Merge pull request #56 from Arkant/feature/weather

Feature/weather

PodstDO committed:
feature/admin-modal, init component

PodstDO committed:
fix/home, fix preloader

Arkant committed:
* add validation for email and password length
* add password match check
* fix email save to profile

pseudopilot committed:
feature/sign-up,

* change logic for parsing datas after singning-up

Arkant committed:
Merge pull request #57 from Arkant/fix/home

fix/home, fix preloader

Arkant committed:
Merge pull request #58 from Arkant/fix/sign-up

 * add validation for email and password length

Arkant committed:
fix notification on invalid user login or password

Arkant committed:
Merge pull request #59 from Arkant/feature/sign-up

feature/sign-up,

Commit:
feature/edit-profile,
1) add defaul user name to header.html
2) improve function saveToCurrentUser
3) add alert message for email (edit-profile.html)
4) add ability to choose ava

Arkant committed:
fix error catch in service

Arkant committed:
Merge branch 'development' into fix/sign-in

Commit:
feature/edit-profile,
fix merge conflict

Commit:
feature/edit-profile,
add ability to save ava to Fyrebase

pseudopilot committed:
Merge pull request #61 from Arkant/feature/edit-profile

Feature/edit profile

Arkant committed:
Merge pull request #60 from Arkant/fix/sign-in

fix notification on invalid user login or password

PodstDO committed:
feature/admin-modal, update:
- add template
- improve modal controller
- improve admin controller
- create share-admin-factory to sync data between admin page and modal

PodstDO committed:
Merge branch 'development' into feature/admin-modal

Commit:
feature/edit-profile,
fix bug with saving to firebase

Commit:
Merge branch 'development' of github.com:Arkant/AngularJSMiniApp into feature/edit-profile

Arkant committed:
fix email validation

Arkant committed:
fix error throw when response is not given

PodstDO committed:
feature/admin-modal, update:
- improve template
- edit getCheckedUserDealsFromFirebase method
- use getCheckedUserDealsFromFirebase method

Arkant committed:
Merge pull request #62 from Arkant/feature/edit-profile

Feature/edit profile

Vladyslav-Shvorobuk committed:
Merge pull request #63 from Arkant/fix/bugs

fix email validation

Arkant committed:
Merge pull request #64 from Arkant/feature/admin-modal

Feature/admin modal

PodstDO committed:
fix/admin, add preloader

Vladyslav-Shvorobuk committed:
Merge pull request #65 from Arkant/fix/admin

fix/admin, add preloader

PodstDO committed:
Merge pull request #66 from Arkant/feature/admin

feature/admin, add change user role functionality

Commit:
feature/edit-profile
change function onFileChanged

Commit:
feature/edit-profile
fixed merge conflict

ElenaBorisenkoO committed:
feature/admin, add change user role functionality

PodstDO committed:
feature/weather, update:
- add buttons
- edit weatherAPIservice
- add method getForecast ro controller
- add method getLocal to controller

PodstDO committed:
Merge branch 'development' into feature/weather

pseudopilot committed:
feature/adminRights,

* add modules firebase functions and firebase tools

pseudopilot committed:
feature/adminRights,

* add modules firebase functions and firebase tools

pseudopilot committed:
feature/adminRights,

* add modules firebase functions and firebase tools

pseudopilot committed:
feature/adminRights,

* add modules firebase functions and firebase tools

PodstDO committed:
feature/weather, current location:
- add module angular-geolocation
- add geolocation service
- add method getLocal to the WeatherController
- add method getCoordinatesForecast to the weatherAPIService
- use method getCityForecast instead of getForecast in the weatherAPIService

PodstDO committed:
feature/weather, add LocalStorageModule

PodstDO committed:
feature/weather, add ng-Autocomplete module

davidkin committed:
update/converter, delete class custom-select and add my class select

davidkin committed:
udpate/converter, add custom style for select

Commit:
feature/edit-profile,
1) change setTimeout to $timeout
2) change apply to applyAsync

motifej committed:
Merge pull request #69 from Arkant/update/converter

update/converter

Arkant committed:
rewrite signin controller on classes

Arkant committed:
rewrite signin controller on classes

pseudopilot committed:
feature/adminRights,

* add admin method for changing password

pseudopilot committed:
feature/adminRights,

* add admin method for changing password

pseudopilot committed:
feature/adminRights,

* add admin method for changing password

pseudopilot committed:
feature/adminRights,

* add admin method for changing password

PodstDO committed:
feature/weather, update:
- edit template
- use ngAutocomplete by google
- add methods getForecastForSelected and setWeatherData to controller
- weatherAPIservice: code refactor, use only getForecast method

pseudopilot committed:
feature/adminRights,

* delete extra line from .gitignore

pseudopilot committed:
feature/adminRights,

* delete extra line from .gitignore

pseudopilot committed:
feature/adminRights,

* delete extra line from .gitignore

pseudopilot committed:
feature/adminRights,

* delete extra line from .gitignore

Arkant committed:
fix/issues rewrite html
* rewrite form error messages with ng-message
* remove ng-classes
* delete unused component file

Arkant committed:
fix/issues rewrite html
* rewrite form error messages with ng-message
* remove ng-classes
* delete unused component file

pseudopilot committed:
feature/adminRights,

* remove extra modules from package.json

pseudopilot committed:
feature/adminRights,

* remove extra modules from package.json

PodstDO committed:
feature/weather, edit template

davidkin committed:
update/admin, delete extra divs and fix ng-checked

davidkin committed:
update/admin, update table styles for admin

davidkin committed:
update/admin, change overflow-x from hidden to auto

pseudopilot committed:
fix/userObjects,

* add userId to it userObjects

PodstDO committed:
feature/weather, ngStorage:
use ng-storage instead of angular-local-storage
create localStorage service for setting and getting coordinates
add getSavedCoordinates ethod to the weather controller

PodstDO committed:
Merge branch 'development' into feature/weather

PodstDO committed:
feature/weather, rewrite controller to ES6 style

PodstDO committed:
feature/weather, remove useless angular-local-storage module

PodstDO committed:
feature/weather, use destructing in setWeatherData

PodstDO committed:
feature/weather, code refactoring:
- remove setWeatherData from the controller
- replace it to weatherAPI service

ElenaBorisenkoO committed:
feature/table-directive, add directive for table

PodstDO committed:
Merge pull request #72 from Arkant/update/admin

update/admin

PodstDO committed:
Merge branch 'development' into feature/weather

Commit:
feature/edit-profile,
1) rewrite profile, edit-profile to classes
2) delete alert-message success and styles for it
3) delete $timeout

Arkant committed:
add angular-toastr package

Arkant committed:
fix toastr:
* styled toastr message
* add message to html
* add message func to controller

Arkant committed:
fix toastr:
* styled toastr message
* add message to html
* add message func to controller

Arkant committed:
remove unused tosastr string

PodstDO committed:
feature/weather, replace modules from index.module to index.vendor:
- angularjs-geolocation
- ng-autocomplete
- ng-storage

PodstDO committed:
feature/weather, rename getSavedCoordinates to getForecastForSavedLocation

PodstDO committed:
feature/weather, rename getLocal to getForecastForLocal

Commit:
feature/edit-profile,
add directive on-file-change

Commit:
Merge branch 'development' of github.com:Arkant/AngularJSMiniApp into feature/edit-profile

PodstDO committed:
feature/weather, use coords object instead of lat and long variables

PodstDO committed:
feature/weather, edit selectedLocation value

Commit:
feature/edit-profile,
rewrite function createFormInfo and saveToCurrentUser using destracture

Arkant committed:
fix toaster:
* move toastr import to vendorfile
* move toastr style to bootstrap file
* add AS syntax to sign-in controller
* rewrite sign in function, delete template string

Arkant committed:
rewrite quotes, move toastr css to vendor file

Arkant committed:
Merge pull request #70 from Arkant/fix/issues

Fix/issues

Commit:
feature/edit-profile,
1) delete ng-hide, add ng-message to edit-profile.html
2) make more readable code in edit-profile.html

PodstDO committed:
feature/weather, load Google Places API key from a .env file

PodstDO committed:
Merge branch 'development' into feature/weather

Commit:
feature/edit-profile,
1) delete ng-class
2) relocate profile from scope to this

Commit:
Merge branch 'development' of github.com:Arkant/AngularJSMiniApp into feature/edit-profile

pseudopilot committed:
feature/adminRights,

* change the way of importing firebase to the project,

* fix bug with changing role of user,
* add clearign rootScope objects after log-out,
* change object of users to array of users

pseudopilot committed:
feature/adminRights,

* change the way of importing firebase to the project,

* fix bug with changing role of user,
* add clearign rootScope objects after log-out,
* change object of users to array of users

pseudopilot committed:
feature/adminRights,

* change the way of importing firebase to the project,

* fix bug with changing role of user,
* add clearign rootScope objects after log-out,
* change object of users to array of users

pseudopilot committed:
feature/adminRights,

* conflicts fixed

davidstoff committed:
feature/menu, commenting extra tags and create menu by ng-repeat. But work not correctly

davidstoff committed:
feature/menu,
* delete function realization and create class;
* create variable menu;
* create method menuShow;

ElenaBorisenkoO committed:
feature table-directive, add admin part
pass callbacks to directive

Commit:
feature/edit-profile,
add default ava

ElenaBorisenkoO committed:
feature/table-directive, add constants
remove unnecessary html from admin.html
add temperary variables for admin contriller

ElenaBorisenkoO committed:
resolve merge conflicts

Commit:
feature/edit-profile,
delete unnecessary ckeck

Commit:
feature/edit-profile,
delete unnecessary link

Vladyslav-Shvorobuk committed:
Feature/edit profile (#67)

* feature/edit-profile
change function onFileChanged

* feature/edit-profile,
1) change setTimeout to $timeout
2) change apply to applyAsync

* feature/edit-profile,
1) rewrite profile, edit-profile to classes
2) delete alert-message success and styles for it
3) delete $timeout

* feature/edit-profile,
add directive on-file-change

* feature/edit-profile,
rewrite function createFormInfo and saveToCurrentUser using destracture

* feature/edit-profile,
1) delete ng-hide, add ng-message to edit-profile.html
2) make more readable code in edit-profile.html

* feature/edit-profile,
1) delete ng-class
2) relocate profile from scope to this

* feature/edit-profile,
add default ava

* feature/edit-profile,
delete unnecessary ckeck

* feature/edit-profile,
delete unnecessary link

PodstDO committed:
feature/weather, use google api key from env

PodstDO committed:
Merge branch 'feature/weather' of github.com:Arkant/AngularJSMiniApp into feature/weather

pseudopilot committed:
Merge branch 'development' into feature/adminRights

ElenaBorisenkoO committed:
feature/table-directive, replace this.scope with this

ElenaBorisenkoO committed:
feature/table-directive, return constants to headcontrollers
and admin controller

pseudopilot committed:
feature/adminRights,

* removes extra comment lines

pseudopilot committed:
feature/adminRights,

* removes extra comment lines

pseudopilot committed:
feature/adminRights,

* removes extra comment lines

pseudopilot committed:
feature/adminRights,

* change listOfUsers back to Object

pseudopilot committed:
:Merge branch 'feature/adminRights' of https://github.com/Arkant/AngularJSMiniApp into feature/adminRights

pseudopilot committed:
feature/adminRights,

* change single semicolons to double in index.vendor

pseudopilot committed:
Merge pull request #71 from Arkant/feature/adminRights

Feature/admin rights

Arkant committed:
fix/sign-up add toaster, remove scope usage, add red border on input error

Arkant committed:
add max length for firstname and lastname

Arkant committed:
remove comments

Arkant committed:
Merge branch 'development' into fix/sign-up

Arkant committed:
Merge pull request #76 from Arkant/fix/sign-up

Fix/sign up

pseudopilot committed:
Merge branch 'development' into fix/userObjects

ElenaBorisenkoO committed:
Merge pull request #75 from Arkant/feature/table-directive

Feature/table directive

Commit:
feature/edit-profile,
1) add toastr-success
2) delete some unnecessery ng-messages
3) make profile.html more understandebale
4) change default ava
5) add new favicon

Commit:
feature/edit-profile,
fix merge conflict

Commit:
feature/edit-profile,
1) delete profile.controller
2) relocate toaster.success to sync-data.service.js

ElenaBorisenkoO committed:
feature/table-directive, move table template to
separate html file

Commit:
feature/edit-profile,
relocate then with toaster from sync-data.service to edit-profile.controller

ElenaBorisenkoO committed:
feature/table-directive, return old variant to admin controller

Arkant committed:
Merge pull request #77 from Arkant/feature/edit-profile

Feature/edit profile

ElenaBorisenkoO committed:
feature/table-directive, fix merge conflicts

davidkin committed:
Merge pull request #73 from Arkant/fix/userObjects

fix/userObjects,

ElenaBorisenkoO committed:
fix/admin-page, make admin page asynchronous

ElenaBorisenkoO committed:
feature/table-directive, edit admin controller,
edit bootstap templates for admin page,
remove unnecessary console logs

motifej committed:
Merge branch 'development' into feature/weather

pseudopilot committed:
feature/cloudFunctions,

* add functions for changing user passwords and deleting all authorized users

pseudopilot committed:
feature/cloudFunctions,

* add functions for changing user passwords and deleting all authorized users

PodstDO committed:
Merge pull request #74 from Arkant/feature/weather

Feature/weather

pseudopilot committed:
feature/privateRouting,

* move states description from modules to index.routes.js
* add preloading functions to resolver.provider.js
* add constants roles.USER and roles.ADMIN
* add data.role for all the states in index.routes to set access permissions
* add transitions listener in index.run.js
* remove extra mocks

ElenaBorisenkoO committed:
fix/admin-page, replace console.log with .log

ElenaBorisenkoO committed:
fix/admin-page, move operations with data to service

pseudopilot committed:
Merge branch 'development' into feature/cloudFunctions

ElenaBorisenkoO committed:
fix/admin-page, replace password template with real login

ElenaBorisenkoO committed:
fix/admin-page, replace strings user/admin with constants

ElenaBorisenkoO committed:
fix/admin-page, change bad names, remove comments and logs

Arkant committed:
Merge pull request #79 from Arkant/feature/cloudFunctions

feature/cloudFunctions,

pseudopilot committed:
fix/userRole,

* add constants for roles USER and ADMIN
* add objects destructurizations
* add small fixes

Commit:
feature/edit-profile,
1) add phone.directive.js
2) add new validation
3) add phoneFilter

davidkin committed:
Merge branch 'development' of github.com:Arkant/AngularJSMiniApp into feature/menu

davidkin committed:
feature/menu, update variable this.menu

davidkin committed:
feature/menu, add dropdown menu

PodstDO committed:
fix/admin-page, improve modal with user history:
- improve controller
- use custom-table directive

davidkin committed:
feature/menu, return currentUser after geting response from firebase. It needs for menu show

PodstDO committed:
fix/admin-page, rewrite admin modal controller to the ES6 style with class

davidkin committed:
feature/menu, delete extra else if

ElenaBorisenkoO committed:
fix/admin-page, fix bug with user role change

PodstDO committed:
fix/environment-file, use config.js instead of env.js (#82)

pseudopilot committed:
Merge pull request #81 from Arkant/feature/privateRouting

Feature/private routing

pseudopilot committed:
Merge branch 'development' into fix/userRole

pseudopilot committed:
Merge pull request #84 from Arkant/fix/userRole

fix/userRole,

Commit:
Merge branch 'development' of github.com:Arkant/AngularJSMiniApp into feature/edit-profile

Commit:
feature/edit-profile,
fix bug, add uid to createFormInfo and saveToCurrentUser

ElenaBorisenkoO committed:
Merge branch 'development' into update/converter

ElenaBorisenkoO committed:
update/converter, add toastr for success deals and warning
add zeroing inputs with amounts after success deal
add styles for button buy

Vladyslav-Shvorobuk committed:
Merge pull request #86 from Arkant/update/converter

update/converter, add toastr for success deals and warning

PodstDO committed:
fix/admin-page, rewrite resolve provider to service

PodstDO committed:
fix/admin-page, resolve merge conflicts

davidkin committed:
Merge branch 'development' of github.com:Arkant/AngularJSMiniApp into feature/menu

Commit:
feature/edit-profile,
1) delete phoneFilter
2) rewrite saveToCurrentUser and createFormInfo

Commit:
Merge branch 'development' of github.com:Arkant/AngularJSMiniApp into feature/edit-profile

Arkant committed:
fix/html fix header display, button style, table display

davidkin committed:
feature/menu, add method deafaultMenu() and start use role constants

PodstDO committed:
fix/admin-page, use resolver SERVICE instead of PROVIDER

davidkin committed:
feature/menu, change ng-show to ng-if

Vladyslav-Shvorobuk committed:
Merge pull request #85 from Arkant/feature/edit-profile

Feature/edit profile

PodstDO committed:
fix/admin-page, remove config.js from repo

Vladyslav-Shvorobuk committed:
Merge pull request #87 from Arkant/fix/html

fix/html fix header display, button style, table display

davidkin committed:
Merge branch 'development' into feature/menu

davidkin committed:
Merge pull request #83 from Arkant/feature/menu

feature/menu

PodstDO committed:
fix/admin-page, use env variables for setting API key for weather widget

pseudopilot committed:
Merge pull request #91 from Arkant/fix/admin-page

Fix/admin page

ElenaBorisenkoO committed:
fix/home-page, 1. rewrite home controller in class style
2. remove useless filter
3. add home controllerAs

ElenaBorisenkoO committed:
Merge pull request #92 from Arkant/fix/home-page

fix/home-page

Commit:
fix/main,
delete page main, set sign-in like start page

Commit:
Merge branch 'development' of github.com:Arkant/AngularJSMiniApp into fix/main

pseudopilot committed:
Merge pull request #93 from Arkant/fix/main

Fix/main

pseudopilot committed:
fix/deleteTransactionsPage,

* delete extra page because now we use it as a component

pseudopilot committed:
Fix/add log out functionality (#95)

* fix/changeUserIdName,

* change userId variable name to uid

* fix/addLogOutFunctionality,

* add function logOut to header.controller and put it in template

* fix/addLogOutFunctionality,

* add some code optimizations

* fix/addLogOutFunctionality,

* change logOut from using auth service in header to using event emit
* add sign-up preloader to resolver
* add some styles
* add link to footer

pseudopilot committed:
Merge pull request #96 from Arkant/fix/deleteTransactionsPage

fix/deleteTransactionsPage,

motifej commented on src/app/components/weather/weather.controller.js:
```diff
@@ -0,0 +1,60 @@
+        .then(data => {
+          this.localStorageService.setCoordinates(coords);
+          this.weatherData = data;
+          this.tableShow = true;
```
we don't need this variable

DenisDO replied:
Why? The table with forecast data has this attribute `ng-show="weatherCtrl.tableShow"`, because the table is hidden when this one is empty or data is unavailable.

PodstDO replied:
Done ✔

motifej commented on src/app/components/weather/weather.controller.js:
```diff
@@ -0,0 +1,60 @@
+        });
+    }
+
+    getForecastForLocal() {
```
refactor this function! this is not correct way to work with promise

PodstDO replied:
Done ✔

motifej commented on src/app/components/weather/weather.module.js:
```diff
@@ -0,0 +1,17 @@
+
+import WeatherComponent from './weather.component';
+import './weather.scss';
+import projectConfigVars from '../../../../config';
```
too strange name - just 'config'

PodstDO replied:
Done ✔

motifej commented on src/app/core/services/authentication.service.js:
```diff
@@ -0,0 +1,59 @@
+          .then(function() {
+            $location.path('/home')
+          })
+          .catch(function(error) {
```
why do you need this section(catch) if you do nothing here, just repeat native behavior?

pseudopilot replied:
Later, in sign-up.controller, we use toaster that accept the result of this promise and show it on the screen.
We return only mistake here, so, only error message will be shown by toaster.
If we didn't use catch section we would have throw error without any notifications.

motifej commented on src/app/core/services/authentication.service.js:
```diff
@@ -0,0 +1,59 @@
+            $rootScope.currentUser = user;
+            $location.path('/home');
+          })
+          .catch(function(error) {
```
why do you need this section(catch) if you do nothing here, just repeat native behavior?

pseudopilot replied:
Later, in sign-in.controller, we use toaster that accept the result of this promise and show it on the screen.
We return only mistake here, so, only error message will be shown by toaster.
If we didn't use catch section we would have throw error without any notifications.

motifej commented on src/app/core/services/geolocation.service.js:
```diff
@@ -0,0 +1,13 @@
+    .service('geolocationService', function (geolocation) {
+        'ngInject';
+
+        this.getCoordinates = () => geolocation.getLocation().then(data => ({
```
use destructure then(({ coords })=> ({

PodstDO replied:
Done ✔

motifej commented on src/app/core/services/sync-data.service.js:
```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-console */
```
remove this line

PodstDO replied:
Done ✔

motifej commented on src/app/index.firebase.config.js:
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-console */
```
remove this line

PodstDO replied:
Done ✔

motifej commented on src/app/index.module.js:
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```
remove this line

PodstDO replied:
Done ✔

motifej commented on src/app/pages/sign-in/sign-in.html:
```diff
@@ -0,0 +1,36 @@
+                </div>
+                <div class="mx-auto">
+                    <h4 class="mb-3">Not a member yet?</h4>
+                    <a class="btn-lg btn-primary text-decoration-none" href="../#!/sign-up"><i class="fas fa-user"></i> Sign up now!</a>                
```
it is not correct, use router special attr to go to some route

PodstDO replied:
Done ✔

PodstDO committed:
Refactoring/app (#98)

* refactoring/app, remove importing of transactions page (unviable page)

* refactoring/app, remove useless variable tableShow, use weatherCtrl.weatherData.temp instead

* refactoring/app, rename projectConfigVars to config

* refactoring/app, use destruction in getCoordinates method in geolocationService

* refactoring/app, refactor getForecastForLocal method in weather controller

* refactoring/app, remove:
- eslint-disable
- console logs
- mocks service
- useless footer controller

* refactoring/app, sign up router:
- remove href from "Sign Up" button in Sign In page
- add goToSignUp method to controller

* refactoring/app, use special router attribute instead of ng-click with callback

* refactoring/app, remove useless $state from sign-in controller

* refactoring/app, remove useless footer controller
